flight/goto.py

```diff
@@ -0,0 +1,92 @@
+"""
+File containing the move_to function responsible
+for moving the drone to a certain waypoint and stopping there for 15 secs
+"""
+
+import asyncio
+from mavsdk import System
+import logging
+
+
+async def move_to(
+    drone: System, latitude: float, longitude: float, altitude: float, fast_mode: bool
+) -> None:
+    """
+    This function takes in a latitude, longitude and altitude and autonomously
+    moves the drone to that waypoint. This function will also auto convert the altitude
+    from feet to meters.
+    Parameters
+    ----------
+    drone: System
+        a drone object that has all offboard data needed for computation
+    latitude: float
+        a float containing the requested latitude to move to
+    longitude: float
+        a float containing the requested longitude to move to
+    altitude: float
+        a float contatining the requested altitude to go to (in feet)
+    fast_mode: bool
+        a boolean that determines if the drone will take less time checking its precise location
+        before moving on to another waypoint. If its true, it will move faster, if it is false,
+        it will move at normal speed
+    """
+
+    # converts feet into meters
+    altitude = altitude * 0.3048
```

Comment:
Should create a new variable for this rather than reassigning the parameter variable